wifiphisher/pywifiphisher.py

```diff
@@ -721,7 +722,7 @@ def start(self):
         if args.presharedkey:
             self.access_point.set_psk(args.presharedkey)
         if args.internetinterface:
-            self.access_point.set_internet_interface(args.presharedkey)
+            self.access_point.set_internet_interface(args.internetinterface)
```

Comment:
I think it is a typo? since the method called set_internet_interface but the parameter is `presharedkey` ?